solutions/silver/cses-1619.mdx

```diff
@@ -244,6 +245,64 @@ public class Customers {
 ```
 
 </JavaSection>
+<PySection>
+
+<Warning>
+
+Due to Python's high constant factor, this code may TLE on a couple test cases.
+
+</Warning>
+
+```py
+times = []
+for _ in range(int(input())):
+	start, end = map(int, input().split())
+	times.append((start, 1))
+	times.append((end, -1))
+
+times.sort()
+
+curr_ppl = 0
+max_ppl = 0
+
+for t in times:
+	curr_ppl += t[1]
+	max_ppl = max(max_ppl, curr_ppl)
+
+print(max_ppl)
+```
+
+Since all arrival and leaving times are distinct, we can store if a given time is an arrival or departure, and sort all the arrival and leaving times in one array. This saves a bit of time by removing the need to sort a list of tuples, and passes all cases.
```

Comment:
(optional)

```suggestion
Since all arrival and leaving times are distinct, we can store if a given time is an arrival or departure, and sort all the arrival and leaving times in one array.
```

this line is kinda unnecessary tbh

Reply:
fair